trustfall_core/src/resources/test_data/parse_errors/invalid_filter_argument_name.parse-error.ron

```diff
@@ -0,0 +1,6 @@
+Err(OtherError(
+    "Filter argument names must only contain ASCII alphanumerics or underscore characters: containsåäö",
+    Pos(
+        line: 4,
+        column: 58,
+)))
```

Comment:
```suggestion
)))

```

Reply:
Probably possible to configure your editor to add trailing newlines automatically. When they aren't added, some of the unix tools don't work correctly.